exasol_advanced_analytics_framework/event_context/event_context_base.py

```diff
@@ -0,0 +1,30 @@
+from abc import ABC, abstractmethod
+from typing import Union, List
+from exasol_data_science_utils_python.preprocessing.sql.schema.column import \
+    Column
+
+
+class EventContextBase(ABC):
+    def __init__(self, ctx):
+        self.__ctx = ctx
```

Comment:
While I see it, the base doesn't know the ctx object , only the implementation UDFEventContext for the UDFs knows it. The reason is, that we might want to replace the UDFEventContext